app/javascript/components/patient/monitoring_actions/CaseStatus.js

```diff
@@ -200,6 +209,26 @@ class CaseStatus extends React.Component {
             </React.Fragment>
           )}
           {this.state.modal_text !== '' && <p>{this.state.modal_text}</p>}
+          {this.state.monitoring_option === 'End Monitoring' && (
+            <div>
+              <Form.Group controlId="monitoring_reason">
+                <Form.Label>Please select reason for status change:</Form.Label>
+                <Form.Control as="select" size="lg" className="form-square" onChange={this.handleChange} defaultValue={'Meets Case Definition'}>
+                  <option></option>
+                  {this.props.monitoring_reasons.map((option, index) => (
+                    <option key={`option-${index}`} value={option}>
+                      {option}
+                    </option>
+                  ))}
+                </Form.Control>
+              </Form.Group>
+              <Form.Group controlId="reasoning">
+                <Form.Label>Please include any additional details:</Form.Label>
+                <Form.Control as="textarea" maxLength={MAX_NOTES_LENGTH} rows="2" onChange={this.handleChange} />
+                <Form.Label className="notes-character-limit"> {MAX_NOTES_LENGTH - this.state.reasoning.length} characters remaining </Form.Label>
```

Comment:
this needs to be the `character-limit-text` class name - styling isn't there

Reply:
also should not be label - should be a div

Reply:
Thanks! I fixed it in the other file, but forgot about this one.